source/content/drush-import.md

```diff
@@ -1,37 +1,46 @@
 ---
 title: Importing Drush Site Archives with Terminus
-description: Import a Drupal Drush site archive using the Terminus, the Pantheon CLI tool.
+description: Import a Drupal Drush site archive using Terminus, the Pantheon CLI tool.
 cms: "Drupal"
 categories: [get-started]
 tags: [migrate, terminus, drush]
 ---
+
 One of the easiest ways to move an existing Drupal site to Pantheon is to import a [Drush archive file](https://drushcommands.com/drush-8x/core/archive-dump/) using our [Terminus command-line interface](/terminus). This automates the packaging of the existing installation, improving the chances of success.
 
+Follow the steps below carefully to ensure that you import your Drupal site correctly.
+
 ## Before You Begin
 
-There are a few things you'll need in order to make this work:
+1. Create a Pantheon account with at least one free Dev site slot open. 
+
+   - A [Pantheon account is free](https://dashboard.pantheon.io/register), and if you need an extra Dev site to try this out, just ask and we'll be happy to grant you one.
 
-1. A Pantheon account with at least one free Dev site slot open. A [Pantheon account is free](https://dashboard.pantheon.io/register), and if you need an extra Dev site to try this out, just ask and we'll be happy to grant you one.
-2. A working local Drush installation that is up to date with 7.x or 8.x stable.
-3. Drush access to your existing Drupal site.
+1. Verify that you have Drupal 7 with Drush 8 if you have a non-Composer managed site *or* Drupal 9 with [Drush 11.1.1 or higher](https://www.drush.org/latest/install/) if you have a Composer-managed site.
```

Comment:
I might phrase this differently.

"If you have a Drupal 7 site, verify that you are using Drush 8. Note that Composer-managed sites are not supported."

"If you have a Drupal 9 site, verify that you are using Drush 11. Note that **only** Composer-managed sites are supported."

Maybe we can point other, unsupported configurations back to the "manually migrate" pages?